src/test/test.cpp

```diff
@@ -237,6 +237,12 @@ getTestConfig(int instanceNumber, Config::TestDbMode mode)
         thisConfig.INVARIANT_CHECKS = {
             "(?!EventsAreConsistentWithEntryDiffs).*"};
 
+        // Run all invariants on every ledger in tests, but not for persistent
+        // BucketListDB as it's too slow
+        thisConfig.ALWAYS_RUN_SNAPSHOT_FOR_TESTING =
```

Comment:
We'll probably need to find some middle ground in tests to make sure it doesn't take too long (seems like recent runs are getting close to ~1hr again), but that can be addressed in a follow-up.

Reply:
While this particular invariant is a little redundant, the in-memory snapshot invariant would be valuable to run on every ledger in tests, as we don't really have any invariant on per-commit state there (assuming we drop https://github.com/stellar/stellar-core/pull/4996). My suspicion is this check is not very expensive in most cases, but is very expensive on a small handful of tests. The majority of tests have trivial BucketList state, but the tests that don't usually close many ledgers, such as the longer running BucketList suit of tests.